website/src/pages/docs/guides/_meta.json

```diff
@@ -4,6 +4,7 @@
   "svelte": "Svelte / Kit",
   "flutter-freezed": "Dart/Flutter",
   "graphql-server-apollo-yoga": "Apollo Server / GraphQL Yoga",
+  "graphql-server-apollo-yoga-with-server-preset": "Apollo Server / GraphQL Yoga with Server Preset",
```

Comment:
I manually added this line. Without this, the guide title does not show up correctly on the left-hand side. 

Is this the right place? I'm not sure if this file is generated or not :)

Reply:
yes it is fine, we are using nextra 😇 see https://nextra.site/docs/guide/organize-files#_metajson